pkg/webui/console/containers/event-split-frame/index.js

```diff
@@ -79,14 +103,8 @@ const EventSplitFrameInner = ({ children }) => {
       )}
       {isActive && !isOpen && (
         <div className={style.openButton}>
-          <Button
-            icon={IconLayoutBottombarExpand}
-            tooltip={m.expandEventPanel}
-            tooltipPlacement="left"
-            onClick={() => setIsOpen(true)}
-            secondary
-            small
-          />
+          <LiveDataTutorial setIsOpen={setIsOpen} setTutorialSeen={setTutorialSeen} seen={seen} />
+          <LiveDataButton setIsOpen={setIsOpen} />
```

Comment:
I think it's better to still use the `<Button />` component here and the add a `className` which changes the styling according to the design instead of re-creating a button component.